### ISSUE TITLE
There is no way to include all files in manifest

### ISSUE THREAD

**Issue by Pinpickle** [smysnk/gulp-rev-all#165]
After #151 landed, file extensions are strictly opt-in to include them in the manifest. As some of the discussion in the PR mentioned, this is unintuitive as files are renamed but not included in the manifest without any explanation.

The default behaviour was to include all file types in the manifest, and people were happy with that. I'm proposing we bring that default behaviour back with some way of opting in to the behaviour that #151 introduced.

**Comment by tSte** [comment]
I agree. I just installed and used this plugin. I thought it is a bug...

**Comment by joeskeen** [comment]
Agreed.  I understand adding the option to limit the files in the manifest, but there does need to be a way to disable that filter.  I ended up having to get a list of all extensions to pass it in... not a great solution IMHO.

```js
var path = require('path');
var gulp = require('gulp');
var revAll = require('gulp-rev-all');
var globExpand = require('glob-expand');

gulp.task('revision', () => {
  var src = 'built/**/*.*';
  var extensions = globExpand(src).map(path.extName);
  return gulp.src(src)
    .pipe(revAll({
      includeFilesInManifest: extensions
    })
});
```

Wouldn't it be nicer if you could just do this?
```
.pipe(revAll({
  includeFilesInManifest: '*' // or null or some other value
})
```

Are you open to a pull request for this?  Looking at the code I think it could be a one-liner.

**Comment by niftylettuce** [comment]
If someone submits a PR for this I will tip them on Paypal.

**Comment by niftylettuce** [comment]
It would be great to have globby-like support

**Comment by niftylettuce** [comment]
One could patch this as follows:

Modify <https://github.com/smysnk/gulp-rev-all/blob/7fc61344df3b4377bf54b70d938cda8771096ebb/revisioner.js#L404-L407> such that:

```js
    // Add only specific file types to the manifest file
    if (Array.isArray(this.options.includeFilesInManifest) && this.options.includeFilesInManifest.indexOf(ext) !== -1) {
      this.manifest[pathOriginal] = pathRevisioned;
    }
```
